Source/Core/Subtitles/Helpers.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+#include <string>
+#include <Common/CommonTypes.h>
+#include <Common/StringUtil.h>
+#include <VideoCommon/OnScreenDisplay.h>
+#include <picojson.h>
+#include "WebColors.h"
+
+namespace Subtitles
+{
+
+void Info(std::string msg)
+{
+  OSD::AddMessage(msg, 2000, OSD::Color::GREEN);
+  INFO_LOG_FMT(SUBTITLES, "{}", msg);
+}
+void Error(std::string err)
+{
+  OSD::AddMessage(err, 2000, OSD::Color::RED);
+  ERROR_LOG_FMT(SUBTITLES, "{}", err);
+}
+
+u32 TryParsecolor(picojson::value raw, u32 defaultColor)
+{
+  if (raw.is<double>())
+    return raw.get<double>();
+  else
```

Comment:
I'm all in for more syntax than needed :)
Before PR I trimmed bunch of curlies on one-liners because I saw such style all around the code, I'll have to do those lint style checks.